### ISSUE TITLE
Make dependency obvious

### ISSUE THREAD

**Issue by AntouanK** [AtomLinter/linter-eslint#82]
Just a suggestion, could you make more obvious ( or automatic ) the dependency on the `Linter` package?

For the last 6 months I installed/uninstalled `linter-eslint` 3-4 times, because it never worked.
Only today I realised that I have to go and install manually the `Linter` package.
Or maybe it's just me...

**Comment by iam4x** [comment]
@AntouanK from the README.md

```
Installation

Linter package must be installed in order to use this plugin. If Linter is not installed, please follow the instructions here.
```

**Comment by AntouanK** [comment]
@iam4x I know, and I should have read it. But every time I went into the atom packages section, and all you see there is the `Install` button.
![screen shot 2015-05-22 at 11 35 31](https://cloud.githubusercontent.com/assets/4569111/7768817/be3a7fc0-0076-11e5-92bd-28081d0281ea.png)

So you click install, and then you wait to see it working, right?
The 4th time I decided to actually read the Readme, but I assumed that if `Linter` is a dependency, atom will install it automatically. It sounds dumb that it doesn't actually.
